packages/manager/cypress/e2e/core/account/personal-access-tokens.spec.ts

```diff
@@ -224,4 +231,107 @@ describe('Personal access tokens', () => {
         cy.findByText('No items to display.').should('be.visible');
       });
   });
+
+  /*
+   * - Uses mocked API requests to confirm disabled states for proxy users
+   * - Confirms that a proxy user cannot create an API token
+   * - Confirms that a proxy user cannot edit (rename) an API token
+   * - Confirms that a proxy user can revoke an API token created for them
+   * - Confirms that token is removed from list after revoking it
+   */
+  it('disables API token creation and editing for a proxy user', () => {
+    const proxyToken: Token = appTokenFactory.build({
+      label: randomLabel(),
+      token: randomString(64),
+    });
+    const proxyUserProfile = profileFactory.build({ user_type: 'proxy' });
+
+    // TODO: Parent/Child - M3-7559 clean up when feature is live in prod and feature flag is removed.
+    mockAppendFeatureFlags({
+      parentChildAccountAccess: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    mockGetProfile(proxyUserProfile);
+    mockGetPersonalAccessTokens([proxyToken]).as('getTokens');
+    mockGetAppTokens([]).as('getAppTokens');
+    mockRevokePersonalAccessToken(proxyToken.id).as('revokeToken');
+
+    cy.visitWithLogin('/profile/tokens');
+    cy.wait([
+      '@getClientStream',
+      '@getFeatureFlags',
+      '@getTokens',
+      '@getAppTokens',
+    ]);
+
+    // Find 'Create a Personal Access Token' button, confirm it is disabled and tooltip displays.
+    ui.button
+      .findByTitle('Create a Personal Access Token')
+      .should('be.visible')
+      .should('be.disabled')
+      .click();
+
+    cy.findByRole('tooltip')
+      .should('be.visible')
+      .within(() => {
+        cy.findByText(
+          'You can only create tokens for your own company.'
+        ).should('be.visible');
+      });
```

Comment:
```suggestion
    ui.tooltip
      .findByText('You can only create tokens for your own company.')
      .should('be.visible');
```
Just pointing out that this exists for the future!

The helper is a little bit more ergonomic, and has the extra advantage of working from inside `within(() => {})` callbacks even though the tooltip itself is often somewhere else in the DOM (not applicable to your situation of course)

Reply:
Nice, thanks! I appreciate the reminder that these UI helpers exist; made that quick change in [3a35491](https://github.com/linode/manager/pull/10109/commits/3a35491e60b64327a3eec23d6c3f2807568adf6e).